app/models/comment.rb

```diff
@@ -0,0 +1,6 @@
+class Comment < ApplicationRecord
+  belongs_to :user
+  belongs_to :post
+
+  validates :Comment, length: { minmum: 7, maximum: 40 }
```

Comment:
- I find this a bit problematic, shouldn't it be ```comment``` instead of ```Comment```, the field name being validated should be the same as the name of the field. Kindly address this issue